IFC4x3/Properties/n/NumberOfOccupiedSlots_0BB9fPQzj1QRSm5RHDh4up/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfOccupiedSlots_0BB9fPQzj1QRSm5RHDh4up" Name="NumberOfOccupiedSlots" UniqueId="0b2c9a59-6bdb-4169-b730-15b44dac4e33" PrimaryDataType="IfcInteger" />
```

Comment:
so i understand this is an occurence property and that creating an occurence pset for one property is overkill.
but isnt this reasonably simple to derive from the model, the number of slots is indeed a product property and should be stated. but isn't the number occupied slots just a count of ports or nested objects. (e.g. a domestic fusebox nests a set of breaker modules)the "occupied slots" values is just a count of the nested modules.